charts/hauler_manifest.yaml

```
@@ -46,7 +46,7 @@ metadata:
   name: kubewarden-not-signed-images
 spec:
   images:
-    - name: ghcr.io/kyverno/policy-reporter-ui:2.4.3
+    - name: ghcr.io/kyverno/policy-reporter-ui:2.4.4
```

```suggestion
    - name: ghcr.io/kyverno/policy-reporter-ui:2.4.3
```

policy-reporter 3.5.0 ships with 2.4.3. Updatecli shouldn't bump this.